pkg/audit/cursor_test.go

```diff
@@ -36,7 +36,7 @@ func TestAuditSegment(t *testing.T) {
 		// change limit in library to 5 in
 		// list api call, default is  0 == 1000 listing
 		//populate metainfo with 10 non-expired pointers of test data
-		tests, cursor, metainfo := populateTestData(t, planet, time.Now().Add(3*time.Second))
+		tests, cursor, mi := populateTestData(t, planet, time.Now().Add(3*time.Second))
```

Comment:
why mi instead of metainfo?  metainfo is clearer to me.

Reply:
same here about the variable/package name conflict

Reply:
it collides with an imported package identifier